packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.styles.ts

```diff
@@ -218,15 +218,15 @@ export const getStyles: IStyleFunction<IDropdownStyleProps, IDropdownStyles> = p
     ],
     caretDown: [
       globalClassnames.caretDown,
-      { color: palette.neutralSecondary, fontSize: FontSizes.small, pointerEvents: 'none' },
+      { color: semanticColors.inputPlaceholderText, fontSize: FontSizes.small, pointerEvents: 'none' },
```

Comment:
not a fan of this, but can't find something more appropriate

Reply:
maybe bodySubtext? (or a menuItem equivalent of it)

Reply:
There is nothing in menu that maps to `neutralSecondary`, but I will change to bodySubtext. Thanks for the suggestion!

Reply:
@cliffkoh this is exactly why I we need designers to help us out with this kind of issues by making a pass through all semantic slots and create redlines for each component in pointing the correct usage of these slots. It might be very possible that ew end up with new slots addition but we should be very careful with it.

Reply:
yeah you might want to add a menu item version of bodySubtext at some point

Reply:
bodySubtext is only going to work while the menuBackground and bodyBackground are the same

Reply:
Should I delay merging this PR? Or at least revert these two lines of more contentious changes?

Reply:
up to you. currently there are no themes that customize semantic slots so this is only a theoretical issue